src/index.html

```diff
@@ -2,16 +2,156 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" 
+      autocomplete="off"
+    >
+      <fieldset >
+        <legend>Personal information</legend>
+        <label class="input1">
+          Surname:
+          <input type="text" name="Surname"
+          required
+          >
+        </label>
+        <label class="input">
+          Name:
+          <input type="text" 
+          name="Name"
+          required
+          >
+        </label>
+        <label class="input">
+          How old are You?
+          <input type="number" name="age" 
+          value="12" 
+          min="1" 
+          max="100" 
+          required
```

Comment:
Add two spaces before each property and make them from a new line (1 property - 1 line).